glide.yaml

```diff
@@ -3,7 +3,7 @@ ignore:
 - go/doc
 import:
 - package: github.com/uber-go/tally
-  version: v2.1.0
+  version: ^v2.1.0
```

Comment:
tally is on version 3, so might be better to do `^3`

Reply:
okay.